### PR TITLE
matlab kid overlay added and readme run instruction fix

### PR DESCRIPTION
**Pull request by Keenan-Nicholson**

### CODE REVIEW

**Comment by Keenan-Nicholson** [comment]
@nint8835

File: pkg/bot/.DS_Store
Comment (jackharrhy) [inline pkg/bot/.DS_Store:1]: add to gitignore

File: pkg/bot/overlays.go
Comment (jackharrhy) [inline pkg/bot/overlays.go:1]: this change just places the image in the corner of the screen no? i thought the vibe was more making it full size as a semi transparent overlay or something

**Review by jackharrhy — CHANGES_REQUESTED** [review]

File: pkg/bot/.DS_Store
Comment (Keenan-Nicholson) [reply pkg/bot/.DS_Store:1]: oh didn't even know this would show up lol

File: pkg/bot/overlays.go
Comment (Keenan-Nicholson) [reply pkg/bot/overlays.go:1]: yeah just throws it on the screen i can do full size transparent i guess

Comment (jackharrhy) [reply pkg/bot/overlays.go:1]: full size transparent feels like more cryptid vibes

File: pkg/.DS_Store
Comment (ranguli) [inline pkg/.DS_Store:1]: add to gitignore too

File: .DS_Store
Comment (ranguli) [inline .DS_Store:1]: geeeeet ignoooooore

File: pkg/bot/.DS_Store
Comment (ranguli) [reply pkg/bot/.DS_Store:1]: GitHub ignore my Nintendo DS store

Comment (Keenan-Nicholson) [reply pkg/bot/.DS_Store:1]: @jackharrhy fix the DS_Store shit im hungry and the gitignore didnt work and idk how to rewrite commit history in gh to remove the commit that had those files added lol

**Review by nint8835 — APPROVED** [review]

**Pull request merged**